### ISSUE TITLE
Not active by default

### ISSUE THREAD

**Issue by fikriauliya** [hypothesis/browser-extension#48]
I use Chrome extension. Every time I want to see the annotations, I have to click Hypothes.is Chrome's button first:

<img width="41" alt="screen shot 2017-08-07 at 5 52 44 pm" src="https://user-images.githubusercontent.com/341555/29023775-487a9658-7b99-11e7-8d72-1adaa9af2497.png">

Only after clicking that, the annotations and the sidebar appear.

Is it intended behavior (why?) or bug?

**Comment by robertknight** [comment]
Hello @fikriauliya - Yes, this is the intended behaviour. @dwhly perhaps you might know the thinking behind this default?

**Comment by katrinleinweber** [comment]
Please do explain this intention. The sidebar sliding away upon every click onto the page or PDF that is being annotated easily doubles the number of clicks for me. Esp. when I'm not the first annotator, I often want to switch between reading, highlighting something myself, appending an annotation to my highlight and reading other people's annotation.

Please consider adding an option (maybe pin button) to keep the sidebar open.

**Comment by dwhly** [comment]
I think there are two conversations happening here.

> Every time I want to see the annotations, I have to click Hypothes.is Chrome's button first.

At present yes, if you turn on the sidebar, by clicking the extension icon, it will stay active on that tab, and for any subsequent pages you browse to on that tab.  That decision was made because we felt generally that people were enabling the annotation capability selectively and we didn't want to have to force it on them for all pages.  We could make this behavior configurable, but we also long ago made a decision to focus on the optimal default behaviors and to skip config heavy admin capabilities.  We're always open to feedback about what people want however.

> The sidebar sliding away upon every click onto the page or PDF that is being annotated easily doubles the number of clicks for me

This is a separate issue.  Many have asked for a way to "pin" the sidebar open, and even to have it not "cover the page" but be side-by-side.  This is a frequent request and one that we should prioritize.

@dawariley ^

**Comment by lkraav** [comment]
> This is a separate issue. Many have asked for a way to "pin" the sidebar open, and even to have it not "cover the page" but be side-by-side. This is a frequent request and one that we should prioritize.

Hi. This is also my first reaction after getting familiar with the workflow.

I don't see a "pin to top" feature issue in the open or closed list. Would it make sense to open one?

**Comment by klemay** [comment]
See also: https://github.com/hypothesis/product-backlog/issues/94

**Comment by klemay** [comment]
Request for this came in via Chrome webstore today - tracked here: https://hypothesis.zendesk.com/agent/tickets/3270

**Comment by klemay** [comment]
Another request: https://hypothesis.zendesk.com/agent/tickets/3280

**Comment by judell** [comment]
https://github.com/hypothesis/product-backlog/issues/175 considers that possibility, but also notes that it'd be ideal to not have to do blacklisting, but rather work out why we conflict with certain sites and solve that.

**Comment by klemay** [comment]
Another request for this came over the weekend: https://twitter.com/nicknikolov/status/1071768432830492679

**Comment by eeril** [comment]
+1 for this request.

I use hypothesis to take notes a lot and I want it to be active-by-default.

**Comment by klemay** [comment]
Hey everyone, this isn't something we're likely to work on soon, as we have lots of other roadmap items to tackle. I did want to mention, though, that you might look into an extensions manager like Extensity to accomplish this for Hypothesis and any other extension you may have installed.

Some of you may already be doing this, but thought I'd say something in case you hadn't tried it!

**Comment by neongreen** [comment]
I would love the ability to make Chrome extension active by default.

"We also long ago made a decision to focus on the optimal default behaviors and to skip config heavy admin capabilities" makes sense as well, though.

@klemay Can Extensity actually _activate_ the H extension? It doesn't look like it can, or at least I haven't found how to accomplish this. It can enable/disable the extension, but that's different.

**Comment by klemay** [comment]
We continue to get feature requests for this, and [some brief internal testing](https://hypothes-is.slack.com/archives/C2BLQDKHA/p1594137032309500?thread_ts=1594071467.304100&cid=C2BLQDKHA) shows that the extension doesn't cause major conflict with tools like Slack and Gmail.

There's still this issue mentioned by Dan above:
 
> we felt generally that people were enabling the annotation capability selectively and we didn't want to have to force it on them for all pages. We could make this behavior configurable, but we also long ago made a decision to focus on the optimal default behaviors and to skip config heavy admin capabilities.

I think it's unlikely that we'll add more config options to the extension any time soon, as we're heavily focused on the LMS app at the moment. However, when we do get to a place where we can focus on the extension, this will be a good candidate.

**Comment by neongreen** [comment]
Btw, what is the LMS app?

**Comment by klemay** [comment]
@neongreen our LMS app brings Hypothesis into LTI-compliant Learning Management Systems like Canvas, Blackboard, etc. More info on that here: https://web.hypothes.is/education/lms/
